ads/_config.js

```diff
@@ -1195,6 +1195,8 @@ const adConfig = jsonConfiguration({
 
   'triplelift': {},
 
+  'digiteka': {},
```

Comment:
I see you have not implemented renderStart, it is up to you but should give you some performance gains.

Reply:
I'll check it out then, thx